README.md

```diff
@@ -101,6 +101,37 @@ poetry install
 
 </details>
 
+<details>
+  <summary>Option 3️⃣ : Docker </summary>
+
+#### Nvidia docker dependency
+  To run the Docker image with GPU support, ensure that you have the NVIDIA Container Toolkit installed. Detailed installation instructions can be found here - https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/latest/install-guide.html.
+
+#### Pull the image and run the container
+To pull our pre-built Docker image and begin using GPUDrive, execute the following command (you may need to prepend sudo, depending on your Docker setup):
+  ```bash
+  docker pull ghcr.io/emerge-lab/gpudrive:latest
+  ```
+
+After pulling the image, you can create and run a new container using the --gpus all flag (omit this flag if your system does not have a GPU):
+  ```bash
+  sudo docker run --gpus all -it --name gpudrive_container gpudrive
+  ```
+
+The Docker image includes all necessary dependencies, along with Conda and Poetry. However, a compilation step is still required. Once inside the container, run:
```

Comment:
Basic question: After doing this, where does the container "live"? How would I be able to `import gpudrive` and use the codebase? Haven't had the time to readup on github packages so excuse me for the basic question

Reply:
So the `docker run` command actually createa a container called `gpudrive_container`. In the current command, you'll immediately be taken into the container and in the gpudrive repository with conda and poetry installed. 
You can exit the container by running `exit` and to access the container again (if it is already running) you can run `sudo docker exec -it gpudrive_container /bin/bash`. In case the container is not running, you can start the container first using `sudo docker start gpudrive_container`.

Let me know if you think this needs to be added to the readme. I didnt add these details because its basic docker usage, in that it can be found in any docker tutorial.

Reply:
Thanks for elaborating on that. Yes, could you please add this information?